README.md

```diff
@@ -22,25 +22,26 @@ This extension contributes the following settings:
 
 ## Known Issues
 
-Right now, the detection for the `@autometrics` decorator is fairly simplistic. This means that the detection will sometimes fail if you use multiple decorators on a single function (the autometrics decorator should be right above the function definition).
+If you mix tabs and spaces in your document, the detection of the autometrics might fall as it is assumed that the whole document is indented the same way
```

Comment:
That seems like a strange limitation. Can't you just look for both?

Reply:
The issue with mixing tabs and spaces is that depending on the platform a tab equals either 4 or 8 spaces. In general I would say that mixing them is bad practice.

Reply:
Sure. But "be strict in what you produce but liberal in what you accept"

Reply:
I've added code that tries to guess what the indentation you're aiming for